src/main/java/io/split/android/client/service/splits/SplitKillTask.java

```diff
@@ -43,6 +49,7 @@ public SplitTaskExecutionInfo execute() {
             splitToKill.changeNumber = mKilledSplit.changeNumber;
 
             mSplitsStorage.updateWithoutChecks(splitToKill);
+            mEventsManager.notifyInternalEvent(SplitInternalEvent.SPLIT_KILLED_NOTIFICATION);
```

Comment:
Javi, check but I don't see that we are handling SPLIT_KILLED_NOTIFICATION event in the SplitsEventsManager

Reply:
Good catch Doc. Done!